openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -35,8 +35,12 @@ function download {
     index=$(( i+1 ))
     file="${files[$i]}.yaml"
     target_file="$target_dir/$index-$file"
-    url="https://github.com/knative/$component/releases/download/$version/$file"
 
+    if [[ "${component}" == "serving" ]]; then
```

Comment:
Instead of doing a `serving` switch here, we should probably use the `versions.le` function from `common.bash` to determine if the version is less than `1.0.0` and adjust the tag accordingly.

Though, I guess, this code will go away once eventing moves on anyway so it might be fine as-is.